source-browser.html

```diff
@@ -72,16 +135,13 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           name: topic,
           repoUrl: `https://github.com/${ORGANIZATION}/apertium-${topic}`,
         }));
-
```

Comment:
Please undo the deletion of all these new lines.

Reply:
This time only one change had made in the old script! so done :)